### PR TITLE
Add facebook-login indicator

### PR DESCRIPTION
**Pull request by mimi89999**

### CODE REVIEW

File: indicators/facebook-login.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+references:
+  - https://urlscan.io/result/36b94762-6afb-40b5-9d7d-576656b97d57/
+  - https://urlscan.io/result/cf1d63b6-a55f-4675-8925-6edfbf0027e2/
+  - https://urlscan.io/result/8b88051f-5115-44cd-b9bf-452266997a0a/
+
+detection:
+  ogTags:
+    html|contains: '<meta property="og:title" content="Log in to Facebook | Facebook">'
+  piwik:
+    requests|contains: 'danielhar.containers.piwik.pro'
```
Comment (bradleyjkemp) [inline indicators/facebook-login.yml:13]: I think it'd be higher signal here to include more of this unique-looking URL:
```suggestion
    requests|contains: 'danielhar.containers.piwik.pro/9dda3b8f-d3cd-49b0-9bab-fc06cb34f389'
```

**Review by bradleyjkemp — COMMENTED** [review]
Hey @mimi89999, thanks for the rule 💙

I just have two small suggestions:
* There's a nice unique-looking ID in the request to `danielhar.containers.piwik.pro` which looks useful to include in the rule
* Can we rename this file to something like `facebook-9dda3b8f`—there's lots of Facebook phishing kits so we need some way to distinguish them and for the moment we're just appending some short unique-ish string from the kit itself

**Comment by mimi89999** [comment]
Hello,
Thanks for your suggestions.

> There's a nice unique-looking ID in the request to danielhar.containers.piwik.pro which looks useful to include in the rule

I also saw it, but I think that including it is not a good idea. If their account gets suspended, they might just register a new one and the ID will change.

> Can we rename this file to something like facebook-9dda3b8f

I saw that Discord IOKs have such a hash, but I couldn't find any doc on how it is computed. Could you please document it?

**Comment by IlluminatiFish** [comment]
> > Can we rename this file to something like facebook-9dda3b8f
> 
> I saw that Discord IOKs have such a hash, but I couldn't find any doc on how it is computed. Could you please document it?

I just appended the first seven characters/digits from the page's sha256 hash or if the indiciator is unique enough I just supplied the first seven characters/digits of it instead.

**Comment by bradleyjkemp** [comment]
> > There's a nice unique-looking ID in the request to danielhar.containers.piwik.pro which looks useful to include in the rule
> 
> I also saw it, but I think that including it is not a good idea. If their account gets suspended, they might just register a new one and the ID will change.

I actually think `https://danielhar.containers.piwik.pro/9dda3b8f-d3cd-49b0-9bab-fc06cb34f389/noscript.html` would be a really good indicator: it's high-entropy (so likely to be unique across phishing kits) and it seems to just be used as part of their `<noscript>` fallback so I don't think they'd notice if it broke so wouldn't replace it.

It also gives an easy ID for this rule: `facebook-9dda3b8f` 🙂

**Review by bradleyjkemp — APPROVED** [review]
Thanks for the rule @mimi89999! 💙 

At some point there'll be the option for contributors to get some IOK/Phish Report stickers but I've not figured out shipping yet. If you're interested I'll come back with details once I've figured them out 🙂

**Pull request merged**